src/Command/Import_Command.php

```
@@ -0,0 +1,718 @@
+	}
+
+	/**
+	 * Process the links.
+	 *
+	 * @since 1.0.0
+	 *
+	 * @return void
+	 */
+	public function process_urls(): void {
+
+		$current_count = 0;
```

PHPMD (2.15.0)

Avoid unused local variables such as '$current_count'. (Unused Code Rules)

(UnusedLocalVariable)